modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -105,25 +105,29 @@ public List<NavigationItem> getInfoPanelNavigationItems() {
 			).build());
 	}
 
-	public long getItemToMoveParent(String itemToMoveType)
+	public long getItemToMoveParent()
```

Comment:
We should replace all occurrences of 'item' by 'kbObject' in the parts that depend directly of KB. If the react component is generic, then it may be ok to keep the more generic "item", but in KB classes/JSPs we should stick with 'kbObject' consistently.